packages/react-ui/storybook/stories/atoms/Palette.stories.js

```diff
@@ -59,13 +60,19 @@ const ColorBox = ({ colorVariant, colorName }) => {
   const color = theme.palette[colorVariant];
   const colorValue = colorName ? color[colorName] : color;
   const classes = useStyles();
+  const textRef = useRef();
 
   return (
     <Box>
       <Box className={classes.text}>
         <Typography variant='subtitle1'>{colorName}</Typography>
         <Tooltip title={colorValue} enterDelay={600}>
-          <Typography variant='caption' noWrap className={classes.textValue}>
+          <Typography
+            variant='caption'
+            noWrap
+            className={classes.textValue}
+            ref={textRef}
```

Comment:
why are you adding ref here?

Reply:
Because of this: https://mui.com/material-ui/guides/composition/#caveat-with-refs

Tooltip (and a few others) needs to pass the ref to its children. I had to add the `forwardRef` in Typography to do that because is a custom component.

Without this, I was having this warning in the console:
`Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?`